resources/sts/4.11/sts_installer_privatelink_permission_policy.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "ec2:ModifyVpcEndpointServiceConfiguration",
+		    "route53:ListHostedZonesByVPC",
```

Comment:
This is a new permission where did it come from?

Reply:
@hunterkepley do you know why this permission was added?
Otherwise, I can remove it.

Reply:
@oriAdler Feel free to remove it, I added it without checking if it was needed